gammapy/utils/random.py

```diff
@@ -30,6 +30,14 @@ def sample_sphere(size, lon_range=None, lat_range=None, unit='radians'):
     lon, lat: arrays
         Longitude and latitude coordinate arrays
     """
+    # Correctly interpret units at an early stage
+    if unit in ['rad', 'radian', 'radians']:
```

Comment:
OK. Maybe on a separate pull request.
Issue #283. (for some reason I can't assign it to myself?)
Can you remind me how to make a full-text search on Github?